modules/backend/behaviors/relationcontroller/partials/_button_reorder.htm

```diff
@@ -0,0 +1,14 @@
+<?php $relationName = $this->vars['relationField']; ?>
+<a
+        data-control="popup"
+        data-size="huge"
+        data-handler="onRelationButtonReorder"
+        data-request-data="
+            _reorder_parent_id[<?= e($relationName) ?>]: '<?= e($this->vars['formModel']->id) ?>',
+            _reorder_model[<?= e($relationName) ?>]: '<?= e(addslashes(get_class($this->vars['formModel']))) ?>',
+            _reorder_relation_name: '<?= e($relationName) ?>'
+        "
```

Comment:
@tobias-kuendig I haven't looked at the rest of your code in depth yet, but is this something that would benefit from https://github.com/octobercms/october/pull/5136?

Reply:
I guess the `_reorder_parent_id` and `_reorder_model` fields could benefit from your PR. These are used to fetch the form model, that is being edited. The `_reorder_relation_name` is then used to update the pivot data on the given relation.